components/ChallengeForm.js

```diff
@@ -10,7 +10,7 @@ import {
 } from "../components/StyledComponents";
 
 async function persistAddedChallenge(challenge) {
-  const baseUrl = "http://localhost:3000";
+  const baseUrl = process.env.MONGODB_URI;
```

Comment:
I'm not sure what's happening here: why do save a baseURL in your .env file and call it MongoDB_URI? As far as I can see, you don't have a backend so far. Should we speak about this in person?